# find . -type f -exec fgrep -q LV2UI_Descriptor {} \; \
#                -exec sed -i {} -e 's/const _\?LV2UI_Descriptor/const LV2UI_Descriptor/' \;
#
--- eq10q-2.2.org/.svn/pristine/1b/1bfb215962413197431e0f05fc93daa3e96db807.svn-base
+++ eq10q-2.2/.svn/pristine/1b/1bfb215962413197431e0f05fc93daa3e96db807.svn-base
@@ -37,7 +37,7 @@
 using namespace std;
 #endif
 
-static LV2UI_Handle instantiateDyn_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateDyn_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateDyn_gui Entring... ";
--- eq10q-2.2.org/.svn/pristine/56/56d71d7e5b8ffea0e9116fc5deec5cd32d86a12e.svn-base
+++ eq10q-2.2/.svn/pristine/56/56d71d7e5b8ffea0e9116fc5deec5cd32d86a12e.svn-base
@@ -37,7 +37,7 @@
 using namespace std;
 #endif
 
-static LV2UI_Handle instantiateMidSide_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateMidSide_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
--- eq10q-2.2.org/.svn/pristine/b2/b2639de412118ae094aaaac6d9be2afa06acf590.svn-base
+++ eq10q-2.2/.svn/pristine/b2/b2639de412118ae094aaaac6d9be2afa06acf590.svn-base
@@ -38,7 +38,7 @@
 #endif
 
 
-static LV2UI_Handle instantiateEq10q_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateEq10q_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
--- eq10q-2.2.org/.svn/pristine/ea/ea8c4d0a9fc4136c04533a0b7bc33a4dc70dd1d6.svn-base
+++ eq10q-2.2/.svn/pristine/ea/ea8c4d0a9fc4136c04533a0b7bc33a4dc70dd1d6.svn-base
@@ -38,7 +38,7 @@
 #define BASSUP_GUI_URI "http://eq10q.sourceforge.net/bassup/gui"
 
 
-static LV2UI_Handle instantiateBassUp_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateBassUp_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
--- eq10q-2.2.org/gui/bassup_ui.cpp
+++ eq10q-2.2/gui/bassup_ui.cpp
@@ -38,7 +38,7 @@
 #define BASSUP_GUI_URI "http://eq10q.sourceforge.net/bassup/gui"
 
 
-static LV2UI_Handle instantiateBassUp_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateBassUp_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
--- eq10q-2.2.org/gui/dyn_ui.cpp
+++ eq10q-2.2/gui/dyn_ui.cpp
@@ -37,7 +37,7 @@
 using namespace std;
 #endif
 
-static LV2UI_Handle instantiateDyn_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateDyn_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateDyn_gui Entring... ";
--- eq10q-2.2.org/gui/eq10q_ui.cpp
+++ eq10q-2.2/gui/eq10q_ui.cpp
@@ -38,7 +38,7 @@
 #endif
 
 
-static LV2UI_Handle instantiateEq10q_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateEq10q_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
--- eq10q-2.2.org/gui/midside_ui.cpp
+++ eq10q-2.2/gui/midside_ui.cpp
@@ -37,7 +37,7 @@
 using namespace std;
 #endif
 
-static LV2UI_Handle instantiateMidSide_gui(const _LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
+static LV2UI_Handle instantiateMidSide_gui(const LV2UI_Descriptor *descriptor, const char *plugin_uri, const char *bundle_path, LV2UI_Write_Function write_function, LV2UI_Controller controller, LV2UI_Widget *widget, const LV2_Feature *const *features)
 {
   #ifdef TESTING_EQ10Q
   cout<<"instantiateEq10q_gui Entring... ";
